_explore/queries/repo-Dependencies.gql

```diff
@@ -1,12 +1,24 @@
 query ($ownName: String!, $repoName: String!, $numManifests: Int!, $numDependents: Int!, $pgCursor: String) {
   repository(owner: $ownName, name: $repoName) {
+    languages(first: 100) {
```

Comment:
While it does seem _highly_ unlikely that a given repo would have more than 100 languages in it, there are 500+ possible recognized languages. (GitHub utilizes [github/linguist](https://github.com/github/linguist) for this.) So there isn't an actual guarantee here without pagination.

Could we simply import the `explore/github-data/labRepos_Languages.json` data for the graph to avoid needing to incorporate this here?

Reply:
I actually only need the top language, but when I pulled only 1, it didn't seem to work properly. Other than that, sure, but we would still need to pull the language data from the dependencies.

Reply:
Ah I overlooked the need for the dependency languages too.
If we can request just the top language, or even just a few, that should be fine. 100 seems excessive if we only need 1

Reply:
Yes, I missed the need for "sort". I am running the query again, but it takes over 5000 queries to run.

Reply:
I will push it when it is done. It will definitely reduce the size of the json.